contracts/src/v0.8/test/HeartbeatRequester.t.sol

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: MIT
```

Comment:
All automation code should live within the `automation` folder, also tests. Please review this CLIP for context https://github.com/smartcontractkit/CLIPs/pull/53/files?short_path=bb0fb24#diff-bb0fb24337ec27c317d92f8574b11c738d1e4c284d495c9450507f9dfb8c5b5f

Reply:
Thank you for pointing this out.
HeartbeatRequester is not Automation specific, it is useful for other products/teams. So should they still live in v0.8/test? 
v0.8/test/HeartbeatRequester/BaseTest.t.sol 
v0.8/test/HeartbeatRequester/(rest of HeartbeatRequester test contracts)